.github/workflows/test.yml

```diff
@@ -10,20 +10,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.10"]
```

Comment:
GitHub examples use double quotes in YAML files.

Reply:
Left it unchanged in new commit and completely dropped the Yaml formatter.